base_layer/wallet/src/text_message_service/service.rs

```diff
@@ -394,6 +372,27 @@ impl Service for TextMessageService {
             })?;
 
         self.oms = Some(context.outbound_message_service());
+
+        // Check if the database file exists
+        let mut exists = false;
+        if std::fs::metadata(self.database_path.clone()).is_ok() {
+            exists = true;
+        }
+
+        let connection = SqliteConnection::establish(&self.database_path)
```

Comment:
you check above if it exists, here you assume it exists? Can will this work?

Reply:
Yes, if it exists it opens the file and if it doesn't exist it creates the file. The `exists` flag is used to determine if it is required to run the migration on a newly created file or not.